Jenkinsfile

```diff
@@ -9,7 +9,18 @@ node {
 
   stage('Build') {
     // you should build this repo with a maven build step here
-    echo "hello"
+    withMaven (maven: 'maven3') {
+          sh "mvn package"
+        }
+    echo "hello ended maven"
   }
   // you should add a test report here
-}
+  echo "before Test"
+  junit "target/surefire-reports/*.xml"
+  // stage('Test') {
+  finally {
```

Comment:
This block is redundant since you have already invoked junit on line 19.  No deduction